.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Question
+    url: https://github.com/weiameili/Pensela/discussions/new
+    about: Please ask and answer questions here.
```

Comment:
This is the Contact links section which doesn't seem like contact links since I added a new discussion link.